src/uct/tcp/sockcm/sockcm_iface.c

```diff
@@ -122,9 +133,132 @@ static uct_iface_ops_t uct_sockcm_iface_ops = {
     .iface_get_address        = uct_sockcm_iface_get_address
 };
 
+static ucs_status_t uct_sockcm_iface_process_conn_req(uct_sockcm_ctx_t *sock_id_ctx)
+{
+    uct_sockcm_iface_t      *iface      = sock_id_ctx->iface;
+    uct_sockcm_conn_param_t *conn_param = &(sock_id_ctx->conn_param);
+
+    ucs_debug("process conn req conn_param = %p, conn_param->length = %ld", 
+              conn_param, conn_param->length);
+    iface->conn_request_cb(&iface->super.super, iface->conn_request_arg, sock_id_ctx, 
+                           conn_param->private_data, conn_param->length);
+
+    return UCS_OK;
+}
+
+static void uct_sockcm_iface_recv_handler(int fd, void *arg)
+{
+    uct_sockcm_ctx_t *sock_id_ctx = (uct_sockcm_ctx_t *) arg;
+    ucs_status_t status;
+    ssize_t recv_len;
+
+    if (sock_id_ctx->recv_len == -1) {
```

Comment:
looks like recv_len represents socket state?
can we do it by conn_state instead?